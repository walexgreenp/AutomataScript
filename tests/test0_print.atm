PRINT: Testing print;
PRINT: This should exit properly;
PRINT:;
