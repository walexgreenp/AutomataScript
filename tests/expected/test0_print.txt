------------------
* AutomataScript *

- Testing print 
- This should exit properly 
- 
------------------
